draft-barnes-acme.md

```diff
@@ -323,6 +323,12 @@ recoveryToken (optional, string):
 agreement (optional, string):
 : A URI referring to a subscriber agreement or terms of service provided by the server (see below).  Including this field indicates the client's agreement with these terms.
 
+authorizations (optional, string):
+: A URI from which a list of authorizations granted to this account can be fetched via a GET request.  The result of the GET request MUST be a JSON array of strings, where each string is the URI of an authorization belonging to this registration.
```

Comment:
We should clarify: Does this include both pending and final authorizations? Does it include expired ones?